lib/src/ignore.dart

```diff
@@ -448,7 +448,8 @@ _IgnoreParseResult _parseIgnorePattern(String pattern, bool ignoreCase) {
     if (nextChar == null) break;
     current++;
     if (nextChar == '*') {
-      if (peekChar() == '*') {
+      final peek = peekChar();
```

Comment:
I think calling `peekChar` multiple times is fine...

Reply:
++